functionlab.js

```diff
@@ -0,0 +1,109 @@
+// question 1
+function double(num) {
+    return num * 2;
+}
+console.log(double(4));
+//question 2
+function smallest(num1,num2) {
+    if (num1 > num2){
+        return num2
+    }else {
```

Comment:
The else statement would run if they're equal too

Reply:
thanks for catching that, definitely overlooked that.